src/components/AddressSearch/index.tsx

```diff
@@ -338,11 +362,10 @@ function AddressSearch(
                     ref={containerRef}
                 >
                     <GooglePlacesAutocomplete
-                        disableScroll
                         fetchDetails
                         suppressDefaultStyles
                         enablePoweredByContainer={false}
-                        predefinedPlaces={predefinedPlaces ?? undefined}
+                        predefinedPlaces={filteredPredefinedPlaces}
```

Comment:
NAB: It's probably ok, but does passing an empty array vs undefined change the behavior of this component at all?

Reply:
@pac-guerreiro, bump on this comment